Makefile

```diff
@@ -199,3 +200,7 @@ generate-metallb-manifests:
 validate-metallb-manifests:
 	@echo "Comparing newly generated MetalLB manifests to existing ones"
 	hack/compare-gen-manifests.sh
+
+lint:
```

Comment:
pls add help description for this new target and make sure it shows in make help

Reply:
done